src/openms/source/CHEMISTRY/ModificationsDB.cpp

```diff
@@ -50,10 +50,16 @@ using namespace std;
 
 namespace OpenMS
 {
-  ModificationsDB::ModificationsDB()
+  ModificationsDB::ModificationsDB(OpenMS::String unimod_file, OpenMS::String psimod_file)
   {
-    readFromUnimodXMLFile("CHEMISTRY/unimod.xml");
-    readFromOBOFile("CHEMISTRY/PSI-MOD.obo");
+    if (!unimod_file.empty())
```

Comment:
is this still necessary? or can you just do

```
  readFromUnimodXMLFile(unimod_file);
  readFromOBOFile(psimod_file);
```
is it a good idea to fail here silently if the string is empty? it should actually never be empty - better to call these functions and let them throw an exception if they think that this wont work :-)